ozaria/site/components/play/PageUnitMap/common/UnitMapLevelDot.vue

```diff
@@ -0,0 +1,178 @@
+<script>
+  import { getNextLevelLink } from 'ozaria/site/common/ozariaUtils'
+  export default Vue.extend({
+    props: {
+      levelData: {
+        type: Object,
+        required: true
+      },
+      courseId: {
+        type: String,
+        default: undefined
+      },
+      courseInstanceId: {
+        type: String,
+        default: undefined
+      },
+      campaignId: {
+        type: String,
+        default: undefined
+      },
+      codeLanguage: {
+        type: String,
+        default: undefined
+      }
+    },
+    data: () => ({
+      levelType: '',
+      levelStatus: '',
+      levelIcon: {},
+      concepts: ''
+    }),
+    computed: {
+      isCutsceneLevel: function () {
+        if (this.levelData.type !== 'intro') {
+          return false
+        }
+        const introContent = this.levelData.introContent
+        if (_.isArray(introContent)) {
+          return introContent.length === 1 && introContent[0].type === 'cutscene-video'
+        }
+        else if (_.isObject(introContent)) {
+          const language = this.codeLanguage || 'python'
+          const content = introContent[language] || []
+          return content.length === 1 && content[0].type === 'cutscene-video'
+        }
+        return false
+      },
+      levelDotPosition: function () {
+        let position = {
+          left: this.levelData.position.x + '%',
+          bottom: this.levelData.position.y + '%'
+        }
+        return position
+      },
+      levelDotClass: function () {
+        return {
+          locked: this.levelData.locked,
+          next: this.levelData.next,
+          'has-tooltip': this.levelData.next || !this.levelData.locked
+        }
+      },
+      getPlayLevelLink: function () {
```

Comment:
Yeah, here its calling `getNextLevelLink` but it does not involve any lookup/network call. Instead, it puts together a string for the next level URL based on various conditions.
I dont think we have a naming guidance overall, but I have been using `get` for anything that's used to get anything.

Reply:
Now that I think about it, I think it makes sense to remove `get` from here since its a computed property and used as an href for an anchor tag. Thanks for asking about this! 👍